AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -160,6 +163,10 @@ + (void)signOut {
 
 - (void)signIn {
   if (self.clientApplication == nil || self.identityConfig == nil) {
+    if ([[MS_Reachability reachabilityForInternetConnection] currentReachabilityStatus] == NotReachable) {
+      MSLogError([MSIdentity logTag], @"User sign-in failed. Internet connection is down.");
+      return;
+    }
     MSLogDebug([MSIdentity logTag], @"signIn is called while it's not configured or not in the foreground, waiting.");
     self.signInDelayedAndRetryLater = YES;
```

Comment:
Same comment as Android, sorry for not having updated the tasks in due time, but we also forbid calling in background now, this field can be removed altogether.